### PR TITLE
Restrict uEnv.txt (ostree) import 

### PR DESCRIPTION
**Pull request by ricardosalveti**

### CODE REVIEW

**Review by igoropaniuk — APPROVED** [review]
LGTM

**Review by mike-scott — APPROVED** [review]
LGTM

File: meta-lmp-base/recipes-bsp/u-boot/u-boot-ostree-scr-fit/boot-common.cmd.in [hunk 1]
```diff
@@ -3,7 +3,7 @@ setenv fio_msg "FIO:"
 
 # uEnv handling
 setenv bootcmd_resetvars 'setenv kernel_image; setenv bootargs; setenv kernel_image2; setenv bootargs2'
-setenv bootcmd_otenv 'run bootcmd_resetvars; ext4load ${devtype} ${devnum}:${rootpart} ${loadaddr} /boot/loader/uEnv.txt; env import -t ${loadaddr} ${filesize}'
+setenv bootcmd_otenv 'run bootcmd_resetvars; ext4load ${devtype} ${devnum}:${rootpart} ${loadaddr} /boot/loader/uEnv.txt; env import -t ${loadaddr} ${filesize} kernel_image bootargs kernel_image2 bootargs2'
 setenv bootcmd_bootenv 'setenv bootfirmware_version; ext4load ${devtype} ${devnum}:2 ${loadaddr} ${ostree_root}/usr/lib/firmware/version.txt; env import -t ${loadaddr} ${filesize} bootfirmware_version'
```
Comment (mike-scott) [inline meta-lmp-base/recipes-bsp/u-boot/u-boot-ostree-scr-fit/boot-common.cmd.in:7]: Do you want to limit this `env import` as well?

**Review by mike-scott — CHANGES_REQUESTED** [review]

File: meta-lmp-base/recipes-bsp/u-boot/u-boot-ostree-scr-fit/boot-common.cmd.in (hunk 1 above)
Comment (mike-scott) [reply meta-lmp-base/recipes-bsp/u-boot/u-boot-ostree-scr-fit/boot-common.cmd.in:7]: Ah nevermind!  It's already limited.  (Needed to scroll over more)

**Review by mike-scott — APPROVED** [review]

**Review by Tim-Anderson — APPROVED** [review]
LGTM

**Pull request merged**